src/core.c/CompUnit/RepositoryRegistry.pm6

```diff
@@ -53,6 +48,21 @@ class CompUnit::RepositoryRegistry {
         }
     }
 
+    method !register-custom-lib-repo(
```

Comment:
For consistency this should be `method !register-custom-lib-repository`, even if it is a private method.

Reply:
Addressed in https://github.com/rakudo/rakudo/pull/4738/commits/8e034779d41b65b75fcad7c497906e085b48fa15